Client/Frontend/Home/LogoHeader/HomeLogoHeaderCell.swift

```diff
@@ -39,6 +39,7 @@ class HomeLogoHeaderCell: UICollectionViewCell, ReusableCell {
 
     // MARK: - Variables
     private var userDefaults = UserDefaults.standard
```

Comment:
Also, looking at this code, I'm not seeing `userDefaults` used anywhere but the declaration, so we can probably remove that as well, if it's not needed.